docs/feature-store/feature-sets.md

```diff
@@ -29,13 +29,16 @@ See also [Ingest data using the feature store](#ingest-data-fs)
 
 Create a new {py:class}`~mlrun.feature_store.FeatureSet` with the base definitions:
 
-* **name**&mdash;The feature set name is a unique name within a project. 
-* **entities**&mdash;Each feature set must be associated with one or more index column. When joining feature sets, the entity is used as the key column.
-* **timestamp_key**&mdash;(optional) Used for specifying the time field when joining by time.
-* **engine**&mdash;The processing engine type:
+* **name** &mdash; The feature set name is a unique name within a project. 
+* **entities** &mdash; Each feature set must be associated with one or more index column. When joining feature sets, the entity or the relation is considered as the keys columns.
```

Comment:
When joining feature sets, the key columns are determined by the the relations field if exist, and otherwise by the entities